src/shell/js/composeui-fdc3/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@morgan-stanley/composeui-fdc3",
+  "version": "0.1.0",
+  "description": "FDC3 DesktopAgent implementation for Compose UI",
```

Comment:
Nitpick: we usually have "ComposeUI" as one word

Reply:
I took this form from some other place, it's pretty inconsistent all around - something we should fix :)